### ISSUE TITLE
[Attention] OpenMMLab Codecamp 🥇🥈🥉

### ISSUE THREAD

**Issue by ice-tong** [open-mmlab/mmeval#55]
![image](https://user-images.githubusercontent.com/32220263/202392444-b72b5ec9-df0c-4501-a525-d7f7b88ac354.png)

## Introduction
Interested in deeply participating in OpenMMLab projects? Want to learn more about OpenMMLab's awesome tools without wasting plenty of time reading docs? The First OpenMMLab Codecamp has begun!! We provide more than a hundred tasks from seventeenth research directions for you to pick. Whether you are a novice in AI or a senior developer, there are suitable tasks for you to participate in. We will provide quick response and full guidance to help you smoothly complete those tasks and upgrade to a core contributor of OpenMMLab. We combined Beijing Super Cloud Center to support with computing power. 
How to participate? 
Select the task you are interested in and submit registration [here](https://openmmlab.com/activity/codecamp/apply). We will inform you in three days if you have enrolled for the tasks, and then you can formulate the task plan with tutor and start development ! Once your PR has passed preliminary review, you can apply for the next task or just wait for the award!
More details: [OpenMMLab Activity page](https://openmmlab.com/activity/codecamp)。

<meta charset="utf-8"><div data-page-id="RBiRd57XbojN22xY2k7cD6RXnmf" data-docx-has-block-data="true"><div>

Task | Discription | Related skills | Difficulty | Credits
-- | -- | -- | -- | --
Accuracy supports more ML framework（JAX） | Support calculated using jax.numpy.ndarray for Accuracy, and add unit test: https://github.com/open-mmlab/mmeval/blob/main/mmeval/metrics/accuracy.py Multiple dispatch：https://mmeval.readthedocs.io/en/latest/design/multiple_dispatch.html | Python、 JAX | Easy | 10
MeanIoU supports more ML framework（JAX） | Support calculated using jax.numpy.ndarray for MeanIoU, and add unit test: https://github.com/open-mmlab/mmeval/blob/main/mmeval/metrics/mean_iou.py#L21 Mutiple dispatch：https://mmeval.readthedocs.io/en/latest/design/multiple_dispatch.html | Python、JAX | Easy | 10
Add a new Metric to MMEval | Select a Metric from the list, add it to MMEval and adapt it to the algorithm library, see the detailed list of tasks in：https://github.com/open-mmlab/mmeval/issues/50 | Python | Medium | 30
Add a new Metric to MMEval | Select a Metric from the list, add it to MMEval and adapt it to the algorithm library, see the detailed list of tasks in：https://github.com/open-mmlab/mmeval/issues/50 | Python | Medium | 30
Add a new Metric to MMEval | Select a Metric from the list, add it to MMEval and adapt it to the algorithm library, see the detailed list of tasks in：https://github.com/open-mmlab/mmeval/issues/50 | Python | Medium | 30
Add a new Metric to MMEval | Select a Metric from the list, add it to MMEval and adapt it to the algorithm library, see the detailed list of tasks in：https://github.com/open-mmlab/mmeval/issues/50 | Python | Medium | 30
Add a new Metric to MMEval | Select a Metric from the list, add it to MMEval and adapt it to the algorithm library, see the detailed list of tasks in：https://github.com/open-mmlab/mmeval/issues/50 | Python | Medium | 30
Add a new Metric to MMEval | Select a Metric from the list, add it to MMEval and adapt it to the algorithm library, see the detailed list of tasks in：https://github.com/open-mmlab/mmeval/issues/50 | Python | Medium | 30
Add a new Metric to MMEval | Select a Metric from the list, add it to MMEval and adapt it to the algorithm library, see the detailed list of tasks in：https://github.com/open-mmlab/mmeval/issues/50 | Python | Medium | 30
Add a new Metric to MMEval | Select a Metric from the list, add it to MMEval and adapt it to the algorithm library, see the detailed list of tasks in：https://github.com/open-mmlab/mmeval/issues/50 | Python | Medium | 30
Add a new Metric to MMEval | Select a Metric from the list, add it to MMEval and adapt it to the algorithm library, see the detailed list of tasks in：https://github.com/open-mmlab/mmeval/issues/50 | Python | Medium | 30
Add a new Metric to MMEval | Select a Metric from the list, add it to MMEval and adapt it to the algorithm library, see the detailed list of tasks in：https://github.com/open-mmlab/mmeval/issues/50 | Python | Medium | 30

Sign Up Here : [application form ](https://openmmlab.com/activity/codecamp/apply)
By the way, we strongly encourage you to publish your experience on social media like medium or twitter with tag "OpenMMLab Codecamp" to share your experience with more developers!
[https://discord.gg/KuWMWVbCcD](https://cdn.vansin.top/open-mmlab/cjsky.jpg)
Discussion group: discord link
Welcome to join the discussion below or in discord. Come to take the challenge, and become a contributor to the OpenMMLab !
